akka-http2-support/src/main/scala/akka/http/scaladsl/Http2.scala

```diff
@@ -60,36 +60,41 @@ class Http2Ext(private val config: Config)(implicit val system: ActorSystem) ext
 
     val effectivePort = if (port >= 0) port else 443
 
-    val serverLayer: BidiFlow[HttpResponse, ByteString, ByteString, HttpRequest, NotUsed] = {
-      val wrappedContext = WrappedSslContextSPI.wrapContext(httpsContext)
-      val tls = http.sslTlsStage(wrappedContext, TLSRole.server)
+    def http2Layer(): BidiFlow[HttpResponse, SslTlsOutbound, SslTlsInbound, HttpRequest, NotUsed] =
```

Comment:
Yeah I see, this triggered the discussion about "stram local place to put things" I assume as well?
So be it for now, but add a comment about this - so someone isn't tempted to "oh, let's make it a val" 😉